Components/Badges/badge.css

```diff
@@ -0,0 +1,39 @@
+.badge {
+    position: relative;
+}
+
+.state-div {
+    position: absolute;
+    bottom: 4px;
+    right: 2px;
+    border-radius: 50%;
+    width: 14px;
+    height: 14px;
```

Comment:
You can use rems here instead of px

Reply:
I was avoiding the going for fractional value of rem like (0.85 rem) so used px.

Reply:
Ok, understoos